src/worker.ts

```diff
@@ -221,8 +221,12 @@ export class Worker {
       connectionStringDocker: `postgresql://postgres:@${container
         .getName()
         .replace("/", "")}:5432/${databaseName}`,
-      networkNameDocker: network.getName(),
-
+      networkDocker: {
+        id: network.getId(),
+        // StartedNetwork.options is private, however we must access it here
+        // using type-safe string index notation for serialization.
+        options: network["options"],
+      },
```

Comment:
🐧  I think we could just do  

```suggestion
      networkDocker: network
```

instead:

Reply:
That's what I tried at fist, but private properties give issues
<img width="781" alt="image" src="https://user-images.githubusercontent.com/721372/208000245-9cabcaa6-5953-4d9b-bd78-3123bd605ee4.png">